h/static/scripts/tests/controllers/share-widget-controller-test.js

```diff
@@ -0,0 +1,144 @@
+'use strict';
+
+const syn = require('syn');
+
+const ShareWidgetController = require('../../controllers/share-widget-controller');
+
+describe('ShareWidgetController', () => {
+
+  const widgetTemplate = `
```

Comment:
Worth adding a note here that this duplicates the `share_widget.html.jinja2` template. Right now duplicating the templates in tests doesn't cause that much pain, but we might want to reconsider in future.